public_html/assets/css/styleheader.css

```diff
@@ -0,0 +1,70 @@
+header {
+	background-color: white;
+	paddind: 20px;
+}
+
+nav.navigationBar{
+	padding-top: 20px;
+}
+
+ul.headernav li {
+	display:inline-block;
+    margin:auto;
```

Comment:
can you fix the in-consistant tabs, it looks like some are tabs and some are spaces. maybe we should all choose one or the other